src/mrpro/data/_CsmData/_inati.py

```diff
@@ -0,0 +1,36 @@
+import torch
+
+from mrpro.utils.window import sliding_window
+
+
+def coil_map_study_2d_Inati(data: torch.Tensor, ks: int, power: int, padding_mode='circular'):
+    """Coil sensitivity maps using the method described in Inati et al. 2004.
```

Comment:
Maybe use a `References` section as recommended in the [official style guide](https://numpydoc.readthedocs.io/en/latest/format.html#references)

Reply:
Yes, good call!
Only problem is that i don't know which paper this is from, as this was the only reference in @fzimmermann89 implementation. I will happily beautify this, if someone could give me a link to the paper :)

Reply:
1. Inati S J, Hansen, M. S. & Peter, K. A solution to the phase problem in adaptvie coil combination. in Proceedings of the 21st Annual Meeting of ISMRM, Salt Lake City, USA, 2672 (2013).
2. Inati, S. & Hansen, M. A Fast Optimal Method for Coil Sensitivity Estimation and Adaptive Coil Combination for Complex Images. in Proceedings of Joint Annual Meeting ISMRM-ESMRMB, Milan, Italy, 7115 (2014).